### PR TITLE
Add aliases to Country.Identifier module

### LINKED ISSUE #13: Adding comments for some countries in Country.Identifier
Hi, yesterday I had a bug where I used `congo` from `Country.Identifier` rather than `congoDemocraticRepublicOfThe`. I assumed these were aliases, since people call The Democratic Republic of Congo "Congo" sometimes, but actually `congo` is the Republic of Congo.

Would you be open to a PR that distinguishes the two in `Country.Identifier` with a comment? Looking at the code, it looks like if you're open to this, the best way to do it would be to add a `comment` column to the CSV, and pull the comment out of that if present.

I haven't considered all the countries, but some other countries that might benefit from comments could be North/South Korea (`koreaDemocraticPeoplesRepublicOf`/`koreaRepublicOf`), `laoPeoplesDemocraticRepublic` (Laos), `holySee` (Rome, The Vatican, Vatican City).

Another way this might be implemented is to pull in the data from `aliases.txt` to generate comments for each country. Something like:

```
-- | Suriname, Dutch Guiana, Netherlands Guiana, Republic of Suriname, Republiek Suriname, Surinam
suriname :: Country
```

This would be more verbose (e.g. Russia has 23 aliases), but might be helpful for people who don't use the English/American names I'm familiar with.

Thoughts?

### PR DESCRIPTION
**Pull request by MaxGabriel**
Closes #13 

Here's what I came up with. I could not show the comment if there's only a single alias, thoughts on the aesthetics of that? Screenshot below:

image

I went with semicolons as separators, because some of the aliases themselves have commas in them

### CODE REVIEW

File: country-code-generation/app/Main.hs [hunk 1]
```diff
@@ -13,7 +13,7 @@ import Data.DisjointSet (DisjointSet)
 import Control.Monad.Trans.Class
 import Data.Foldable (for_)
 import Control.Monad
-import qualified Data.Set as S
+import qualified Data.Set as Set
```
Comment (MaxGabriel) [inline country-code-generation/app/Main.hs]: Renamed to `Set` because `S` was already being used by `Siphon`, but could keep it S without any compiler errors if you prefer that

Comment (chessai) [reply country-code-generation/app/Main.hs]: I prefer what you've done. Thanks for noting it.

**Comment by chessai** [comment]
The only thing that caught my eye as hard to parse was the commas and semicolons. A comma right now indicates a break in the country name, whereas a semicolon indicates a new country to the right of the semicolon. My brain did not parse this correctly on first try, but maybe this is different for others. Perhaps a colon would look better instead of a semicolon?

Thoughts @MaxGabriel @andrewthad?

**Comment by chessai** [comment]
Aside from that, this LGTM.

**Comment by andrewthad** [comment]
I like the semicolons as separators. The only decent alternative would be haddock’s syntax for bulleted lists, but I don’t think there’s anything to be gained by switching. The way I expected user’s to interact with these docs is just to Control+F for the country they’re trying to find, and what you’ve done accomplished that goal.
